astrocut/tests/test_cube_cut.py

```diff
@@ -49,19 +50,28 @@ def checkcutout(cutfile, pixcrd, world, csize, ecube, eps=1.e-7):
     assert len(tab) == ntimes, "{} expected {} entries, found {}".format(cutfile, ntimes, len(tab))
     assert (tab['TIME'] == (np.arange(ntimes)+0.5)).all(), "{} some time values are incorrect".format(cutfile)
 
-    check1(tab['FLUX'], x1, x2, y1, y2, ecube[:, :, :, 0], 'FLUX', cutfile)
-    check1(tab['FLUX_ERR'], x1, x2, y1, y2, ecube[:, :, :, 1], 'FLUX_ERR', cutfile)
+    if product == 'SPOC':
+        # TODO: Modify check1 to take TICA - adjust for TICA's slightly different WCS 
+        # solutions (TICA will usually be ~1 pixel off from SPOC for the same cutout)
+        check1(product, tab['FLUX'], x1, x2, y1, y2, ecube[:, :, :, 0], 'FLUX', cutfile)
+        # Only SPOC propagates errors, so TICA will always have an empty error array
+        check1(product, tab['FLUX_ERR'], x1, x2, y1, y2, ecube[:, :, :, 1], 'FLUX_ERR', cutfile)
     
     # Regression test for PR #6
     assert hdulist[2].data.dtype.type == np.int32
 
     return 
 
 
-def check1(flux, x1, x2, y1, y2, ecube, label, cutfile):
-    """Test one of flux or error"""
+def check1(ffi_type, flux, x1, x2, y1, y2, ecube, label, cutfile):
```

Comment:
`ffi_type` is not accessed in this function

Reply:
The `ffi_type` here is `product` which gets fed from `check_cutout`, so it was working but I've renamed it to `product` to avoid confusion.

Reply:
it can be removed as an input from this function.  ffi_type/product is not accessed.